content/redirects/federalist-system-architecture.html

```diff
@@ -0,0 +1,5 @@
+---
+permalink: /assets/documents/federalist-system-architecture.pdf/index.html
+redirect: /pages/documentation/how-pages-works/
```

Comment:
I'm asking about this one in slack. It might already exist somewhere else, like on our diagrams site at https://diagrams.fr.cloud.gov/